src/index.jsx

```diff
@@ -1,7 +1,24 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import App from './App';
-import * as serviceWorker from './serviceWorker';
+import React from 'react'
+import ReactDOM from 'react-dom'
+import App from './App'
+import { Provider } from 'react-redux'
+import configureStore from './state/store/configureStore'
+import * as serviceWorker from './serviceWorker'
+import axios from "axios";
 
-ReactDOM.render(<App />, document.getElementById('root'));
-serviceWorker.unregister();
+axios.defaults.baseURL = "http://localhost:3000/api";
+
```

Comment:
be sure to update this to heroku before we do the final merge

Reply:
done